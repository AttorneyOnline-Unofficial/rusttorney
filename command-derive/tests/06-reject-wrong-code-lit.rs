use command_derive::*;

#[derive(Command)]
enum Enum {
    #[command(code = 123)]
    Variant
}

fn main() {}
