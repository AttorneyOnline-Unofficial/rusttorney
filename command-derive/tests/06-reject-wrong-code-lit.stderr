error: Only string literal allowed as value in #[command(code = "LIT")]
 --> $DIR/06-reject-wrong-code-lit.rs:3:10
  |
3 | #[derive(Command)]
  |          ^^^^^^^
  |
  = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)
